docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -0,0 +1,362 @@
+
+:::info
+Please send any of your feedback on the guide to the NEW — the doc gets iterative updates.
+:::
+
+## Why is this guide needed
+
+As said before, the default way to bridge an ERC-20 token is to deploy on L2 a non-upgradable token and use the general bridge contract, this guide proposes to implement a more complex solution.
+
+The solution involves deploying dedicated upgradable bridge contracts on L1 and L2 and an upgradable token on L2, all governed by the Lido DAO on L1 ([Aragon Agent contract](https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c)) via a dedicated [governance executor](https://github.com/lidofinance/governance-crosschain-bridges/) contract on L2. This architecture is proposed to provide the following capabilities.
+
+1. Passing data, particularly the wstETH/stETH rate when bridging rebasable stETH.
+2. Revamping the token logic, as stETH is not a general-purpose token but an asset built on top of a living liquid-staking protocol.
+3. Future-proofing the token, for example, to avoid high-cost liquidity migration as Ethereum continues evolving and new standards like [ERC-2612](https://eips.ethereum.org/EIPS/eip-2612)/[ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) are adopted.
+4. Pausing and resuming bridging in an emergency or during upgrades.
+
+## The Lido DAO bridging endpoints recognition
+
+Lido DAO can recognize the bridged wstETH endpoints by means of a signaling snapshot. For example, it has happened for: [Base](https://snapshot.org/#/lido-snapshot.eth/proposal/0x8b35f64fffe67f67d4aeb2de2f3351404c54cd75a08277c035fa77065b6792f4), [zkSync Era](https://snapshot.org/#/lido-snapshot.eth/proposal/0xe35f56a6117599eeb9dbef982613c0545710d91403a828d1fba00ab21d5188f3), [Mantle](https://snapshot.org/#/lido-snapshot.eth/proposal/0x1d38c11b27590ab5c69ca21c5d2545d53b7f5150dada7e05f89d500ede5becad), [Linea](https://snapshot.org/#/lido-snapshot.eth/proposal/0x9382624eeee68a175dd7d1438347dbad4899ba0d2bfcf7c3955f087cb9f5cfc4).
+
+If Lido DAO recognizes the bridged wstETH endpoints, in general, it means:
+- the integration is highlighted on the frontend pages: landing, widget, and ecosystem pages;
+- when/if the dedicated bridging Lido UI is implemented, the network will be included;
+- the contracts are under the Lido's bug bounty program;
+- the contracts get monitored by means of [Lido alerting system](https://github.com/lidofinance/alerting-forta/);
+- unlocks the possibility of marketing campaign and/or liquidity support (the decisions are still up to ProRel and LOL).
```

Comment:
LOL can actually independently decide to incentivize projects on networks that have not been formally accepted yet, so I would remove that.

Reply:
Maybe just rename to smt like `possibility of marketing campaign and/or liquidity support (the decisions are still up to ProRel and LOL)`? Not sure about removal because this item seems quite a notable implication of the recognition

Reply:
Ok, agreed!